ui/app/src/components/PluginManagement/PluginManagement.tsx

```diff
@@ -268,7 +268,7 @@ export const PluginManagement = (props: PluginManagementProps) => {
           {plugins?.length === 0 && (
             <EmptyState
               title={
-                <FormattedMessage id="PluginManagement.emptyList" defaultMessage="There are not plugins installed" />
+                <FormattedMessage id="pluginManagement.emptyList" defaultMessage="There are not plugins installed" />
```

Comment:
same

Reply:
the syntax we are using of all  FormattedMessage is with the first letter in lowercase